### PR TITLE
Code for test case DA_DP_TC068

### PR DESCRIPTION
**Pull request by lqthai07t3**
- Add DataHelper class.
- Code for test case DA_DP_TC068.
- Run code and check.

### CODE REVIEW

**Comment by lqthai07t3** [comment]
image

File: TADashboard/src/main/java/page_objects/DataProfilesGeneralSettingsPage.java [hunk 1]
```diff
@@ -1,12 +1,15 @@
...
 import com.logigear.control.common.imp.Button;
-import com.logigear.control.common.imp.Link;
-import com.logigear.driver.DriverUtils;
+import com.logigear.control.common.imp.ComboBox;
+import com.logigear.control.common.imp.TextBox;
 
 public class DataProfilesGeneralSettingsPage {
     private Button btnNext = new Button("css=input[value='Next']");
     private Button btnFinish = new Button("css=input[value='Finish']");
+    private TextBox txtName = new TextBox("css=#txtProfileName");
```
Comment (tanmle) [inline TADashboard/src/main/java/page_objects/DataProfilesGeneralSettingsPage.java]: how about ID?

File: TADashboard/src/test/java/tests/data_profiles/PositiveDPTests.java [hunk 2]
```diff
@@ -3,20 +3,24 @@
...
 import utils.common.Constants;
+import utils.helper.DataHelper;
 import utils.helper.Logger;
 
 import java.util.ArrayList;
 import java.util.List;
 
 public class PositiveDPTests extends BaseTest {
     List<String> preSetDataProfiles = new ArrayList<>();
+    List<String> nonPresetDataProfiles = new ArrayList<>();
```
Comment (tanmle) [inline TADashboard/src/test/java/tests/data_profiles/PositiveDPTests.java]: you have only one nonpreset, why do you need a List?

**Review by tanmle — CHANGES_REQUESTED** [review]

File: TADashboard/src/main/java/page_objects/DataProfilesGeneralSettingsPage.java (hunk 1 above)
Comment (lqthai07t3) [reply TADashboard/src/main/java/page_objects/DataProfilesGeneralSettingsPage.java]: Yeah, of cource. I will fix it for next commit.

File: TADashboard/src/test/java/tests/data_profiles/PositiveDPTests.java (hunk 2 above)
Comment (lqthai07t3) [reply TADashboard/src/test/java/tests/data_profiles/PositiveDPTests.java]: Yeah. In my old code, I use a List for non-preset data profiles. In this way, I will not use it. Thank you

**Comment by lqthai07t3** [comment]
Run code and check again after updating code.

image

**Review by tanmle — APPROVED** [review]

**Comment by lqthai07t3** [comment]
image

**Review by tanmle — APPROVED** [review]

**Review by MinhQuan119 — APPROVED** [review]

**Pull request merged**